docker_util.py

```diff
@@ -0,0 +1,84 @@
+import json
+import subprocess
+
+from verbose_print import print_v
+
+
+class ImageNotFoundError(Exception):
+    def __init__(self, image):
+        self.image = image
+        self.message = f'The image {image} was not found.'
```

Comment:
I would just inline this in your call to this:

```python
raise ImageNotFound(f'...')
```

then your class can just be:

```
class ImageNotFoundError(ValueError):
    pass
```

You can depend on the default exception printing behaviour to print the message

You don't actually use the structured data here so there's no reason to complicate the code to keep track of it.

Reply:
This does simplify things
Just made it a dumb exception